### PR TITLE
[MIG] product_supplierinfo_for_customer_picking: Migration to 13.0

### PR DESCRIPTION
**Pull request by PabloPaezSheridan**

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
In your PR title and also in last commit message should put "13.0" instead of "14.0" hehe

**Comment by PabloPaezSheridan** [comment]
> In your PR title and also in last commit message should put "13.0" instead of "14.0" hehe

@MiquelRForgeFlow changed, yes that commit is a cherry-pick from the migration to 14.0

File: product_supplierinfo_for_customer_picking/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* Nicola Malcontenti <nicola.malcontenti@agilebg.com>
+* Alex Comba <alex.comba@agilebg.com>
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
+* Aaron Henriquez <ahenriquez@eficent.com>
```
Comment (MiquelRForgeFlow) [inline product_supplierinfo_for_customer_picking/readme/CONTRIBUTORS.rst]:
```suggestion
* Aaron Henriquez <ahenriquez@forgeflow.com>
```

File: product_supplierinfo_for_customer_picking/tests/test_product_supplierinfo_for_customer_picking.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
```
Comment (MiquelRForgeFlow) [inline product_supplierinfo_for_customer_picking/tests/test_product_supplierinfo_for_customer_picking.py]:
```suggestion
# Copyright 2018 Forgeflow <http://www.forgeflow.com>
```

**Review by MiquelRForgeFlow — APPROVED** [review]
Remember to edit the title also 😉

**Comment by PabloPaezSheridan** [comment]
@rousseldenis Hi, can you take a look on this PR? we need this merged for our clients. thank you

**Comment by rousseldenis** [comment]
> @rousseldenis Hi, can you take a look on this PR? we need this merged for our clients. thank you

Usually, the OCA flow needs at least 5 days (if no maintainer approves it) before PR merge.

Could you fix pre-commit ?

**Review by nicomacr — APPROVED** [review]
LGTM

**Comment by PabloPaezSheridan** [comment]
@rousseldenis now is all green :)

File: product_supplierinfo_for_customer_picking/models/stock_move.py [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
+                    )
+                )
+                product_customer_code = customer.product_code
+            move.product_customer_code = product_customer_code
 
     product_customer_code = fields.Char(
-        compute="_compute_product_customer_code",
-        string="Product Customer Code",
-        size=64,
+        compute="_compute_product_customer_code", string="Product Customer Code",
```
Comment (rousseldenis) [inline product_supplierinfo_for_customer_picking/models/stock_move.py]: You can remove string as default value is camelcase from fields name.

**Review by rousseldenis — APPROVED** [review]
Code review. Small comment

**Comment by PabloPaezSheridan** [comment]
@rousseldenis all done

**Comment by PabloPaezSheridan** [comment]
@rousseldenis denis we need this for our clients, is anything pending to merge this? tell me if you need any change

**Comment by rousseldenis** [comment]
Done :smiley: 

> @rousseldenis denis we need this for our clients


I don't know your flows but I suppose you can integrate your branch for deployments?

**Pull request merged**